src/api/index.js

```diff
@@ -215,22 +215,28 @@ export function deleteSkill(payload) {
   });
 }
 
-export function saveSkill(payload) {
-  const url = `${API_URL}/${CMS_API_PREFIX}/createSkill.json`;
-  return ajax.post(url, payload);
+export function updateSkill(payload, urlParam) {
+  const url = `${API_URL}/${CMS_API_PREFIX}/${urlParam}`;
+  return ajax.post(url, payload, {
+    headers: { 'Content-Type': 'multipart/form-data' },
+    isTokenRequired: false,
+  });
 }
 
-export function modifySkill(payload) {
-  const url = `${API_URL}/${CMS_API_PREFIX}/modifySkill.json`;
-  return ajax.post(url, payload);
+export function modifySkill(payload, endPoint) {
```

Comment:
Why is the endPoint kept dynamic?

Reply:
I kept it dynamic so that the `then-able` promise is not repeated again(in SkillCreator in saveClick), saves 80+ lines of code

Reply:
This may save some lines of code, but is confusing. Please use the approach throughout.